modules/stochastic_tools/src/vectorpostprocessors/PolyChaosSobolStatistics.C

```diff
@@ -0,0 +1,195 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+// Stocastic Tools Includes
+#include "PolyChaosSobolStatistics.h"
+
+#include "libmesh/utility.h"
+
+registerMooseObject("StochasticToolsApp", PolyChaosSobolStatistics);
+
+defineLegacyParams(PolyChaosSobolStatistics);
```

Comment:
We don't need to do this for new classes. This is only if someone is inheriting from it already.